app/models/package_manager/maven.rb

```diff
@@ -153,6 +157,12 @@ def self.retrieve_versions(versions, name)
           rescue StandardError
             license_list = nil
           end
+
+          if license_list.blank? && pom.respond_to?("project") && pom.project.locate("parent").present?
```

Comment:
Not sure if I should wrap this in a `begin/rescue` block so it doesn't affect actual license fetching in case there's a POM parsing issue. Mostly hoping to quickly see from the logs if there are cases where the child version is off from the listed parent pom version.